src/components/results/recommendations.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    v-if="(!$x.query.search || $x.noResults) && $x.recommendations.length > 0"
```

Comment:
What happened here? When did the recommendations appearance condition change?

Reply:
they were moved to the main.vue component